frontend/packages/console-app/src/components/cloud-shell/redux/reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Map } from 'immutable';
+import { TerminalAction, Actions } from './action';
+
+export type State = Map<string, any>;
+
+export default (state: State, action: TerminalAction) => {
+  if (!state) {
+    return Map({
+      terminal: { isExpanded: false },
```

Comment:
This should be 
```suggestion
      isExpanded: false,
```
because you already have `terminal` property in the combineReducer.